chaincode/utils.go

```diff
@@ -118,3 +118,37 @@ func GetTxCreator(stub shim.ChaincodeStubInterface) (string, error) {
 
 	return sID.GetMspid(), nil
 }
+
+// String returns a string representation for an asset type
+func (assetType AssetType) String() string {
+	switch assetType {
+	case ObjectiveType:
+		return "Objective"
+	case DataManagerType:
+		return "DataManager"
+	case DataSampleType:
+		return "DataSample"
+	case AlgoType:
+		return "Algo"
+	case TraintupleType:
+		return "Traintuple"
+	case TesttupleType:
+		return "Testtuple"
+	case CompositeTraintupleType:
+		return "CompositeTraintuple"
```

Comment:
Not sure about the case. I think it should start with a lower case at least: `dataManager`, `traintuple`, `compositeTraintuple`,....